linux/SystemdMeter.c

```diff
@@ -219,6 +219,8 @@ static void updateViaExec(void) {
          exit(1);
       dup2(fdnull, STDERR_FILENO);
       close(fdnull);
+      // Use of NULL in variadic function must have a pointer cast.
+      // The NULL constant is not required by standard to have a pointer type.
```

Comment:
Which platform is broken enough to require the below cast?

Reply:
Please split this NULL change into another PR as there's several other places in the source suffering from the issue. Would be better to fix this in one go.

Reply:
Sure. #1018.